Tasks/AzureVmssDeployment/task.loc.json

```diff
@@ -10,15 +10,11 @@
     "Build",
     "Release"
   ],
-  "keywords": [
-    "vmss",
```

Comment:
these are not used here. i had mistakenly put them here. they work only in package.json.